Description: Avoid crash when cancelling a search.
From: https://bugzilla.gnome.org/attachment.cgi?id=187690
Author: Cristian Buzduga <cristianbzdg@gmail.com>
Bug-Ubuntu: https://launchpad.net/bugs/651061
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=618214
Applied-Upstream: yes, in gnome-commander 1.4.
---
 src/gnome-cmd-search-dialog.cc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gnome-commander.orig/src/gnome-cmd-search-dialog.cc
+++ gnome-commander/src/gnome-cmd-search-dialog.cc
@@ -369,7 +369,6 @@ static gpointer perform_search_operation
     data->dir = NULL;
 
     data->search_done = TRUE;
-    gtk_widget_set_sensitive (data->dialog->priv->goto_button, TRUE);
 
     return NULL;
 }
@@ -421,6 +420,7 @@ static gboolean update_search_status_wid
             set_statusmsg (data, msg);
             g_free (msg);
 
+            gtk_widget_set_sensitive (data->dialog->priv->goto_button, data->matches > 0);
             gtk_widget_set_sensitive (data->dialog->priv->search_button, TRUE);
             gtk_widget_set_sensitive (data->dialog->priv->stop_button, FALSE);
 
